JuiceMaker/JuiceMaker/Controller/ChangeStockViewController.swift

```diff
@@ -6,41 +6,102 @@ import UIKit
-    @IBOutlet weak var numberOfPineapple: UILabel!
-    @IBOutlet weak var numberOfKiwi: UILabel!
-    @IBOutlet weak var numberOfMango: UILabel!
-    
-    @IBAction func pushPlusStrawberry(_ sender: Any) {
-        //
+    private var fruitsStock: [Fruits: Int] {
+        return FruitStore.shared.fruitsStock
     }
-    @IBAction func pushMinusStrawberry(_ sender: Any) {
-        //
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        displayStock()
+        initializeSteppers()
+        setUpNavigationItem()
     }
-    @IBAction func pushPlusBanana(_ sender: Any) {
-        //
+    
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(true)
+        center.post(name: Notification.Name.fruitStockChanged, object: nil)
     }
-    @IBAction func pushMinusBanana(_ sender: Any) {
-        //
+    
+    @IBOutlet weak var navigationBar: UINavigationItem!
+    
+    @IBOutlet weak var stockOfStrawberry: UILabel!
+    @IBOutlet weak var stockOfBanana: UILabel!
+    @IBOutlet weak var stockOfPineapple: UILabel!
+    @IBOutlet weak var stockOfKiwi: UILabel!
+    @IBOutlet weak var stockOfMango: UILabel!
+    
+    @IBOutlet weak var strawberryStepper: UIStepper!
+    @IBOutlet weak var bananaStepper: UIStepper!
+    @IBOutlet weak var pineappleStepper: UIStepper!
+    @IBOutlet weak var kiwiStepper: UIStepper!
+    @IBOutlet weak var mangoStepper: UIStepper!
+    
+    @IBOutlet var steppers: [UIStepper]!
+    
+    func displayStock() {
+        if let strawberryStock = fruitsStock[.strawberry],
+           let bananaStock = fruitsStock[.banana],
+           let pineappleStock = fruitsStock[.pineapple],
+           let kiwiStock = fruitsStock[.kiwi],
+           let mangoStock = fruitsStock[.mango] {
+            stockOfStrawberry.text = String(strawberryStock)
+            stockOfBanana.text = String(bananaStock)
+            stockOfPineapple.text = String(pineappleStock)
+            stockOfKiwi.text = String(kiwiStock)
+            stockOfMango.text = String(mangoStock)
+        }
     }
-    @IBAction func pushPlusPineApple(_ sender: Any) {
-        //
+    
+    func initializeSteppers() {
+        for stepper in steppers {
+            if let selectedFruit = identifyRelatedFruit(of: stepper),
+               let initialValue = fruitsStock[selectedFruit] {
+                stepper.transform = stepper.transform.scaledBy(x: 1.0, y: 1.0)
+                stepper.value = Double(initialValue)
+            }
+        }
     }
-    @IBAction func pushMinusPineApple(_ sender: Any) {
-        //
+    
+    func identifyRelatedFruit(of stepper: UIStepper) -> Fruits? {
+        switch stepper {
+        case strawberryStepper: return .strawberry
+        case bananaStepper: return .banana
+        case pineappleStepper: return .pineapple
+        case kiwiStepper: return .kiwi
+        case mangoStepper: return .mango
+        default: return nil
+        }
```

Comment:
```suggestion
    func identifyRelatedFruit(of stepper: UIStepper) -> Fruits? {
        switch stepper {
        case strawberryStepper:
            return .strawberry
        case bananaStepper:
            return .banana
        case pineappleStepper:
            return .pineapple
        case kiwiStepper:
            return .kiwi
        case mangoStepper:
            return .mango
        default:
            return nil
        }
```

개인취향 차이가 있을 수 있겠지만 공식문서 상 사용된 예시코드에서는 이런 식의 개행을 적용 해주었답니다☺️

Reply:
가독성 측면에서 한줄로 case를 작성하는게 좋다고 생각했는데, 공식문서가 그랬었군요!